core/ir/x86/opcode_api.h

```diff
@@ -1614,6 +1615,9 @@ enum {
     /* 1433 */ OP_vpopcntd, /**< IA-32/AMD64 vpopcntd opcode. */
     /* 1434 */ OP_vpopcntq, /**< IA-32/AMD64 vpopcntd opcode. */
 
+    /* Privileged (newer) */
```

Comment:
List the actual feature name instead of "Privileged (newer)": "Supervisor Mode Access Prevention (SMAP)."

Reply:
Done.